pages/index.js

```diff
@@ -2,16 +2,37 @@ import Paragraph from "@/components/Layout/Paragraph";
 import ServiceList from "../components/ServiceList/index";
 import StyledHeadlineTwo from "@/components/Layout/StyledHeadlineTwo";
 import ActionLink from "@/components/Layout/ActionLink";
+import useUser from "@/hooks/useUser";
 
-export default function HomePage() {
+export default function HomePage({ user }) {
+  const { logout } = useUser();
+  if (user) {
+    return (
+      <>
+        <Paragraph>
+          Welcome! On this page, you will find a list of Hotel Tishjama&#x27;s
+          bookable, luxury services! Enjoy your stay.
+        </Paragraph>
+        <p>
+          <ActionLink href="./MyBookings">My Bookings</ActionLink>Welcome,{" "}
+          {user.firstName} {user.lastName}!{" "}
+          <button onClick={logout}>Log Out</button>
```

Comment:
If I am seeing it correctly, only these few lines are changing depending on if the user is logged in or not. Instead of having those two return statements, which are almost the same, you could also just change these few lines with a ternary operator if the user is logged in.